tests/integration-e2e/test_dry_run_workflow.py

```diff
@@ -63,9 +63,17 @@ def test_dry_run_workflow(neurodamus_instance, USECASE3):
 
     nd.run()
 
-    assert 20.0 <= nd._dry_run_stats.cell_memory_total <= 30.0
+    # the following uncommented line should be investigated with:
+    # https://github.com/openbraininstitute/neurodamus/issues/44?issue=openbraininstitute%7Cneurodamus%7C45
+    # it should be max 30 but on mac it consumes a little more. For now I increase the limit to
+    # 35 as a workaround
```

Comment:
couldn't we gate the limit based on the operating system, so that instead of expanding it for everything, it's only expanded for macOS?

Reply:
ok, let me open an issue and another pr for this